modules/ols-0-3-3-release-notes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-3-3-release-notes_{context}"]
+= {ols-long} version 0.3.3
+
+{ols-official} 0.3.3 is now available on {ocp-product-title} 4.15 or later.
```

Comment:
```suggestion
{ols-official} 0.3.3 is now available on {ocp-product-title} 4.15 and later.
```


IMO when specifying the availability, "and" is clearer. But I noticed in the preview that "or" is used consistently. So up to you!

Reply:
If you go with "and", you might want to update other similar instances of "or" too.